doc/src/build/prog-trans-ts-sdk.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Sui Programmable Transaction with the TS SDK
+title: Sui Programmable Transaction Blocks with the TS SDK
 ---
 
-In Sui, all user-initiated transactions are called Programmable Transactions.
+In Sui, all user-initiated transactions are called Programmable Transactions blocks.
 
 ## Get Started
 
 First, make sure that you have the latest TypeScript SDK installed.
 
-We’ll start by constructing a transaction to send Sui. If you are familiar with the legacy transaction types, this is similar to a `paySui` transaction. You can start constructing transactions by importing the `Transaction` class, and constructing it:
+To start, construct a transaction to send Sui. If you are familiar with the legacy transaction types, this is similar to a `paySui` transaction. You can start constructing transactions by importing the `TransactionBlock` class, and constructing it:
 
 ```tsx
-import { Transaction } from "@mysten/sui.js";
-const tx = new Transaction();
+import { TransactionBlock } from "@mysten/sui.js";
+const tx = new TransactionBlock();
 ```
 
 Using this, you can then add commands to this transaction.
```

Comment:
Commands are now called transactions, and the outer thing is a transaction block.

Reply:
@Jordan-Mysten You should probably make the updates to this topic